quiche/src/recovery/mod.rs

```diff
@@ -1510,4 +1510,6 @@ mod tests {
 }
 
 mod congestion;
+#[allow(dead_code)]
+mod gcongestion;
```

Comment:
Verified that code still compiles when we removing this; making this PR essentially a Noop